### DISCUSSION TITLE
Configurable default languages for global translation setup

### DISCUSSION THREAD

**Discussion by Jakob-em** [directus/directus#3762] [Feature Requests]
While I was creating multiple fields for a new data type, I was a bit annoyed that I need to create the translation field (language-translation) for every new field. It took quiet some time to select the two languages I need from the dropdown for each item.

It would be great to have some possibility to automatically create the field needed for the translation without selecting the languages every time.
For example a setting for languages that are automatically created for new fields, so that only the translation has to be filled out by the user.

What do you think?

**Comment by philleepflorence** [comment]
Can this be translated to, or is this also what you mean:

> Default language settings that can also be used when creating new _collections_ and _fields_, as opposed to defaulting always to English?

Is this related to: [Default translation and country codes](https://github.com/directus/directus/discussions/3681)

**Reply by Jakob-em** [reply]
Not quite, 
I meant a **set** of default languages that are automatically created with empty values for new fields so that the user only has to fill out the translations and not select the languages every time.
(This would be really beneficial for projects that translate everything into multiple languages)

Sorry if this sounds confusing 😅 , didn't find a better way to phrase it

**Reply by benhaynes** [reply]
This makes sense to me, basically the admin would configure a few languages that would be used as a "default" for the project, and they would be offered up during field creation, etc. Right?

**Reply by Jakob-em** [reply]
Yes that's what I thought of 👍🏻
I would be happy to come up with an implementation if this feature is desired

**Reply by benhaynes** [reply]
Nice, and thank you! Yeah, let's chat on here/discord to get aligned on exactly how this would work. It will be important to get input from @rijkvanzanten (our tech lead) so we don't go down the wrong path. :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
